controllers/admin/AdminImportController.php

```diff
@@ -1260,161 +1805,180 @@ public function categoryImport($offset = false, $limit = false, &$crossStepsVari
         }
 
-        if (!$valid_link) {
+        if (!$validLink) {
             $this->informations[] = $this->trans('Rewrite link for %1$s (ID %2$s): re-written as %3$s.', array(
                 '%1$s' => $bak,
-                '%2$s' => (isset($info['id']) && !empty($info['id']))? $info['id'] : 'null',
-                '%3$s' => $category->link_rewrite[$default_language_id],
+                '%2$s' => $categoryId ? $categoryId : 'null',
+                '%3$s' => $category->link_rewrite[$defaultLanguageId],
             ), 'Admin.Advparameters.Notification');
         }
         $res = false;
-        if (($field_error = $category->validateFields(UNFRIENDLY_ERROR, true)) === true &&
-            ($lang_field_error = $category->validateFieldsLang(UNFRIENDLY_ERROR, true)) === true && empty($this->errors)) {
-            $category_already_created = Category::searchByNameAndParentCategoryId(
-                $id_lang,
-                $category->name[$id_lang],
+        if (($fieldError = $category->validateFields(UNFRIENDLY_ERROR, true)) === true
+            && ($langFieldError = $category->validateFieldsLang(UNFRIENDLY_ERROR, true)) === true
+            && empty($this->errors)
+        ) {
+            $categoryAlreadyCreated = Category::searchByNameAndParentCategoryId(
+                $idLang,
+                $category->name[$idLang],
                 $category->id_parent
             );
 
             // If category already in base, get id category back
-            if ($category_already_created['id_category']) {
-                $cat_moved[$category->id] = (int)$category_already_created['id_category'];
-                $category->id = (int)$category_already_created['id_category'];
-                if (Validate::isDate($category_already_created['date_add'])) {
-                    $category->date_add = $category_already_created['date_add'];
+            if ($categoryAlreadyCreated['id_category']) {
+                $catMoved[$category->id] = (int)$categoryAlreadyCreated['id_category'];
+                $category->id            = (int)$categoryAlreadyCreated['id_category'];
+                if (Validate::isDate($categoryAlreadyCreated['date_add'])) {
+                    $category->date_add = $categoryAlreadyCreated['date_add'];
                 }
             }
 
             if ($category->id && $category->id == $category->id_parent) {
-                $this->errors[] = sprintf(
-                    $this->trans('A category cannot be its own parent. The parent category ID is either missing or unknown (ID: %1$s).', array(), 'Admin.Advparameters.Notification'),
-                    (isset($info['id']) && !empty($info['id']))? $info['id'] : 'null'
+                $this->errors[] = $this->trans('A category cannot be its own parent. The parent category ID is either missing or unknown (ID: %1$s).',
+                    array(!empty($categoryId) ? $categoryId : 'null'),
```

Comment:
Same comment about translation placeholder. Please use a variable name surrounded by `%%` an add it in the second parameter.